.gitignore

```diff
@@ -5,3 +5,6 @@
 googletest
 __pycache__/
 tmp/
+
+# Vim temp files
```

Comment:
This probably belongs in your personal [global gitignore](https://help.github.com/articles/ignoring-files/).